torch/_inductor/ir.py

```diff
@@ -2219,11 +2219,13 @@ def get_fill_order(self):
                 for r in reads
             ]
             # only consider reads to buffer of same size
+            # ignore StarDeps because they don't contribute stride information
             reads = [
                 sympy_subs(
                     r.index, {v: sympy.Integer(0) for v in reduction_vars if v != 0}
                 )
                 for r in reads
+                if isinstance(r, dependencies.MemoryDep)
```

Comment:
What prompted you to make this change, just curious ? I guess as above you got an error because `StarDeps` doesn't have `index`.

Reply:
Yeah, that's why. I guess previously, ComputedBuffers never had StarDeps.

Reply:
yeah just what you said, trying to get r.index.